nft_ingester/src/parsers/bubblegum.rs

```diff
@@ -181,17 +310,19 @@ async fn handle_bubblegum_instruction<'a, 'b>(
         //      otherwise, it becomes hard to reinsert data on a CancelRedeem
         bubblegum::InstructionName::Redeem => {
             println!("Bubblegum: Redeem");
```

Comment:
You definitely want to save redeemed events in the DB (should function a lot like a Burn)